client/my-sites/customer-home/cards/launchpad/index.tsx

```diff
@@ -5,14 +5,14 @@ import {
 	sortLaunchpadTasksByCompletionStatus,
 	LaunchpadNavigator,
 } from '@automattic/data-stores';
-import { Launchpad, Task, setUpActionsForTasks, ShareSiteModal } from '@automattic/launchpad';
+import { DefaultWiredLaunchpad, Task } from '@automattic/launchpad';
```

Comment:
Pre-existing nit: I believe we should be importing `Task` as a type.
```suggestion
import { DefaultWiredLaunchpad, type Task } from '@automattic/launchpad';
```